doc/release-process.md

```diff
@@ -12,10 +12,34 @@ It is best if the maintainers are present during the release, so they can help e
 
 This process also assumes that there will be no minor releases for old major releases.
 
+We aim to cut a regular release every 3-4 months, approximately twice as frequent as major Bitcoin Core releases. Every second release should be published one month before the feature freeze of the next major Bitcoin Core release, allowing sufficient time to update the library in Core.
+
+## Sanity Checks
+Perform these checks before creating a release:
+
+1. Ensure `make distcheck` doesn't fail.
+```shell
+make distcheck
+```
+2. Check installation with autotools:
+```shell
+dir=$(mktemp -d)
+./configure --prefix=$dir && make install && ls -l $dir/include $dir/lib
+gcc -o ecdsa examples/ecdsa.c -I $dir/include -L $dir/lib/ -l secp256k1 && ./ecdsa
+```
+3. Check installation with CMake:
+```shell
+dir=$(mktemp -d)
+mkdir build && cmake -B build -DCMAKE_INSTALL_PREFIX=$dir && cmake --build build --target install && ls -l $dir/include $dir/lib*
+gcc -o ecdsa examples/ecdsa.c -I $dir/include -L $dir/lib*/ -l secp256k1 && ./ecdsa
```

Comment:
Using `-rpath` linker options helps as well:
```suggestion
gcc -o ecdsa examples/ecdsa.c -I $dir/include -L $dir/lib*/ -l secp256k1 -Wl,-rpath,"$ORIGIN/$dir/lib" && ./ecdsa
```

Tbh, I like this approach the most.

Reply:
I added this variant, but without `$ORIGIN` because `$dir` already specifies a full path.